server/src/main/scala/drt/server/feeds/lgw/LGWForecastFeed.scala

```diff
@@ -0,0 +1,219 @@
+package drt.server.feeds.lgw
+
+import java.io.{ByteArrayOutputStream, File, FileReader}
+import java.nio.file.FileSystems
+import akka.NotUsed
+import akka.actor.{ActorSystem, Cancellable}
+import akka.stream.scaladsl.Source
+import akka.stream.{ActorAttributes, Supervision}
+import com.box.sdk.{BoxFile, BoxFolder, _}
+import drt.server.feeds.lgw.LGWFeed.log
+import drt.shared.Arrival
+import org.apache.commons.lang3.StringUtils
+import org.joda.time.format.{DateTimeFormat, DateTimeFormatter, ISODateTimeFormat}
+import org.slf4j.{Logger, LoggerFactory}
+import services.SDate
+import scala.collection.JavaConversions._
+import scala.collection.immutable.Seq
+import scala.collection.mutable.ListBuffer
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.util.matching.Regex
+import scala.util.{Failure, Success, Try}
+
+class LGWForecastFeed(boxConfigFilePath: String, userId: String, ukBfGalForecastFolderId: String) extends BoxFileConstants {
+  val log: Logger = LoggerFactory.getLogger(getClass)
+  val filePattern = "^[0-9]{8}-LGWArrForecast.csv"
+  val LGW = "LGW"
+  val PORT_FORECAST = "Port Forecast"
+  val regex: Regex = """(([^,^\"])*(\".*\")*([^,^\"])*)(,|$)""".r
+  // Set cache info// Set cache info
+  val MAX_CACHE_ENTRIES = 100
+  val accessTokenCache = new InMemoryLRUAccessTokenCache(MAX_CACHE_ENTRIES)
+  val `dd/mm/yyyy HH:mm`: DateTimeFormatter = DateTimeFormat.forPattern("dd/MM/yyyy HH:mm")
```

Comment:
I don't think I like this way of declaring vals, could we rather call this ddMMYYYHHMMFormat?